db/Controllers/topics.controller.js

```diff
@@ -9,4 +9,18 @@ exports.getTopics = (req, res, next) => {
 
 exports.getAllEndpoints = (req, res, next) => {
   res.status(200).send(allEndpoints)
-};
\ No newline at end of file
+};
+
+exports.getArticleById = (req, res, next) => {
+  const { article_id } = req.params
+
+  if (/^\d+$/.test(article_id) === false) {
+    return res.status(400).send({ status: 400, msg: 'Bad request' });
```

Comment:
This is not necessary as you have error handling middleware in your app.js file. Please remove!